roles/ceilometer-common/templates/etc/serverspec/ceilometer_spec.rb

```diff
@@ -20,8 +20,16 @@
   end
 end
 
-describe process('ceilometer') do
-  its(:user) { should eq 'ceilometer' }
+processes = ['ceilometer-api',
+  'ceilometer-collector',
+  'ceilometer-agent-notification',
+  'ceilometer-agent-central',
+  'ceilometer-alarm-evaluator',
+  'ceilometer-alarm-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'ceilometer' }
+  end
```

Comment:
to be completely honest, I don't know enough about ceilometer to give you a 100% answer. But I did read the ceilometer architecture overview (http://docs.openstack.org/developer/ceilometer/architecture.html) and it seems all of these processes are needed for ceilometer to work.